migrations/000018_create_historical_namespace_recommendation_sets.up.sql

```diff
@@ -0,0 +1,21 @@
+CREATE TABLE IF NOT EXISTS historical_namespace_recommendation_sets(
+   id BIGSERIAL NOT NULL,
+   org_id TEXT NOT NULL,
+   workload_id BIGINT NOT NULL,
+   namespace_name TEXT NOT NULL,
+   cpu_request_current FLOAT,
+   cpu_variation FLOAT,
+   memory_request_current FLOAT,
+   memory_variation FLOAT,
+   monitoring_start_time TIMESTAMP WITH TIME ZONE NOT NULL,
+   monitoring_end_time TIMESTAMP WITH TIME ZONE NOT NULL,
+   recommendations jsonb NOT NULL,
+   updated_at TIMESTAMP WITH TIME ZONE NOT NULL
+);
+
```

Comment:
Are you not considering of creating partition at first place?

Reply:
Could you confirm on this? If this change team is planning near future, please raise a ticket to address it for tracking purpose.

Reply:
If you consider ros namespace/project on our k8s there are multiple pods inside the single project, hence the number of recommendations for such a namespace will be much lower than the number of container recomms.

We can create a card but I don't see a need a partitioning anytime soon. 

Please let me know if still prefer to create the same.